src/app/shared/utils/addFavorites.ts

```diff
@@ -0,0 +1,61 @@
+import { changeFavoriteStatus } from './../../ngrx/actions/pokemons-page/pokemons.actions';
+import { first, map, tap } from 'rxjs/operators';
+import { selectAllFavoritePokemons } from './../../ngrx/selectors/favorite-pokemons/favorite-pokemons.selector';
+import { PokemonsPage } from './../../models/shared/pokemons-page';
+import { select, Store } from '@ngrx/store';
+import { Update } from '@ngrx/entity';
+import { addFavorite, removeFavorite } from '../../ngrx/actions/favorite-pokemons/favorite-pokemons.actions';
+
+export function addFavorites(pokemon: PokemonsPage, store: Store): favoritesAddedResponse {
```

Comment:
I recommend using classes with static functions instead of normal functions, as they are easier to test